src/kbp.ts

```diff
@@ -158,15 +227,28 @@ export default class KBPParser {
 				syllable.text = matches[0];
 
 				// Add the start time of the syllable
-				syllable.start = Math.floor(parseInt(matches[1].trim()) * 10);
+
+				// TODO: Better handling of fixed text. Currently this will just set
```

Comment:
How do you know in KBP when a text is fixed ?

Reply:
It's in the style field. Uppercase is wiped lines, and lowercase is fixed. For example, this line would have style 00 with wiping:

`C/A/3648/4349/0/15/0`

This line would be the same but fixed:

`C/a/3648/4349/0/15/0`

The issue is that in .kbp, a fixed line uses the pre-wipe colors whereas in .ass a line without wiping uses the PrimaryColour, so we need to do one of the following:

1. Create an additional style for fixed that sets the primary color to the pre-wipe color and optionally also remove the wiping
2. Add an override tag for the color on each fixed line
3. Set the wiping to occur after the line is gone (solution I chose)

The advantage to (3) is that if you already have timings on the line and decide you do in fact want to wipe it, you can just change the delay in the initial \k tag. But it's otherwise kind of hacky.

Reply:
You can also surcharge the style color for a line with \c or \2c tag if it's useful https://aegisub.org/docs/latest/ass_tags/, see "Set color"

Reply:
Right, that's what I meant by (2). Do you have any preference between (1) or (2)? (2) would definitely be easier to implement with the way the code is currently structured.

Also, if PrimaryColour and SecondaryColour are the same, visually it doesn't matter if the wipe tags are on the line or not. It would look a little cleaner if we removed them, but leaving them in place would allow wiping to be turned back on if desired by just removing the \\1c tag.

Reply:
In our case, we prefer to have specific styles if possible.
But you are the main user of this kbp2ass converter so feel free to choose what you prefer.